examples/python_boids_spatial3D_wrapped_native/boids_spatial3D.py

```diff
@@ -353,17 +355,29 @@ def inputdata(message_in: pyflamegpu.MessageSpatial3D, message_out: pyflamegpu.M
     # Configure vis
     visualisation.setClearColor(255, 255, 255)
     envWidth = env.getPropertyFloat("MAX_POSITION") - env.getPropertyFloat("MIN_POSITION")
-    INIT_CAM = env.getPropertyFloat("MAX_POSITION") * 1.25
-    visualisation.setInitialCameraLocation(INIT_CAM, INIT_CAM, INIT_CAM)
+    INIT_CAM = env.getPropertyFloat("MAX_POSITION") * 3.0
+    visualisation.setInitialCameraLocation(0, 0, INIT_CAM)
     visualisation.setCameraSpeed(0.001 * envWidth)
     visualisation.setViewClips(0.00001, 50)
-    circ_agt = visualisation.addAgent("Boid")
+    boid_agt = visualisation.addAgent("Boid")
     # Position vars are named x, y, z so they are used by default
-    circ_agt.setForwardXVariable("fx")
-    circ_agt.setForwardYVariable("fy")
-    circ_agt.setForwardZVariable("fz")
-    circ_agt.setModel(pyflamegpu.STUNTPLANE)
-    circ_agt.setModelScale(env.getPropertyFloat("SEPARATION_RADIUS") /3.0)
+    boid_agt.setForwardXVariable("fx")
+    boid_agt.setForwardYVariable("fy")
+    boid_agt.setForwardZVariable("fz")
+    #boid_agt.setModel(pyflamegpu.ARROWHEAD) # Alternative simple 3D model for very large pop sizes
+    boid_agt.setKeyFrameModel("./model/bird_a.obj", "./model/bird_b.obj", "wing_animation")
```

Comment:
The python script can only be executed from the `examples/python_boids_spatial3D_wrapped_native/` directory, i.e. `model` must be a child of the current directory. 

E.g. executing it from a build directory doesn't work:

```
python3 ../examples/python_boids_spatial3D_wrapped_native/boids_spatial3D.py 
Traceback (most recent call last):
  File "/home/ptheywood/code/flamegpu/FLAMEGPU2/build-vis/../examples/python_boids_spatial3D_wrapped_native/boids_spatial3D.py", line 381, in <module>
    visualisation.activate()
  File "/home/ptheywood/code/flamegpu/FLAMEGPU2/build-vis/lib/Release/python/venv/lib/python3.10/site-packages/pyflamegpu/pyflamegpu.py", line 24109, in activate
    return _pyflamegpu.ModelVis_activate(self)
RuntimeError: /home/ptheywood/code/flamegpu/FLAMEGPU2/build-vis/_deps/flamegpu_visualiser-src/src/flamegpu/visualiser/util/Resources.cpp(82): Resources::locateFile(): File './model/bird_a.obj' could not be found!
```

With `pathlib`, it's not hard to get the abs path to the dir containing the script file

```
import pathlib
pathlib.Path(__file__).parent.resolve()
```

E.g. this works, but could probably be done cleaner.
```suggestion
    script_dir = pathlib.Path(__file__).parent.resolve()
    bird_a = str(script_dir / "model/bird_a.obj")
    bird_b = str(script_dir / "model/bird_b.obj")
    boid_agt.setKeyFrameModel(bird_a, bird_b, "wing_animation")
```

Reply:
As an aside, is `model` the best term to use (it's correct, but potentially ambiguous)?